### PR TITLE
fix: running project in local

### PR DESCRIPTION
**Pull request by toannvit**
# Linked issue
- #15 

# Changes
- [x] add fallback url when env variables is not provided
- [x] write instructions to start project in local

# Screenshot
image

image

### CODE REVIEW

File: packages/app/src/components/charts/StatisticsCard.tsx [hunk 1]
```diff
@@ -8,13 +8,16 @@ interface GetIdentifierCountResponse {
   count: number
 }
 
+const apiUrl = process.env.REACT_APP_VERAMO_URL || 'http://localhost:3332/agent'
+const apiKey  = process.env.REACT_APP_VERAMO_API_KEY || 'test123'
```
Comment (hoa-ngo-gs) [inline packages/app/src/components/charts/StatisticsCard.tsx]: Can we place the local values into .env.example and add instruction in README?

File: README.md [hunk 2]
```diff
@@ -13,5 +13,29 @@ If you are an organisation that issues any kind of credential such as a permit,
...
+    pnpm run vckit server # to start veramon server
+```
+
+### To run a local client
+
+```bash
+    navigate to package/app
+    pnpm build # to build the client project
+    pnpm run dev # to start client
+```
```
Comment (hoa-ngo-gs) [inline README.md:44]: @arpentnoir  Should we add a few lines about currently having to create config, dids, VCs, etc. using the CLI? Maybe instructions or references on how to do so?

File: packages/app/src/components/charts/StatisticsCard.tsx (hunk 1 above)
Comment (hoa-ngo-gs) [reply packages/app/src/components/charts/StatisticsCard.tsx]: Also, should we move these to a config file?

**Review by hoa-ngo-gs — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -13,5 +13,29 @@ If you are an organisation that issues any kind of credential such as a permit,
...
+```bash
+    pnpm install # to install packages
+    pnpm build # to build and extract api
+    pnpm run vckit server # to start veramon server
+```
+
+### To run a local client
+
+```bash
+    navigate to package/app
```
Comment (namhoang1604) [inline README.md]: Should be `packages/app`, it's missing `s`

File: README.md [hunk 4]
```diff
@@ -13,5 +13,29 @@ If you are an organisation that issues any kind of credential such as a permit,
 VCs issued by vckit tooling can be verified using any mobile device camera to scan a QR code. This is important so that uptake can remain compatible with today's paper processes. There is no requirement for verifiers of credentials to adopt any new new technology in order to verify a credential. However, if you are an organisation that is likely to be verifying at scale or you wish to extract the digitial data in a credential for use in your business systems then vckit is also for you.  It provides an advanced multi-protocol verification capability that can be integrated with your systems.
 
 # Get Started
+ - This project provide all the utilities needed to create, verify and render veriable credentials
 
-TBC once MVP release 1.0 vckit is available.  We expect the first release in March 2023.
+## Prerequisites
+
+```bash
+    veramo >= 5.2.0
```
Comment (namhoang1604) [inline README.md]: I think we don't need to install veramo. We can use `vckit` command line

File: packages/app/src/components/charts/StatisticsCard.tsx (hunk 1 above)
Comment (toannvit) [reply packages/app/src/components/charts/StatisticsCard.tsx]: moved those to env file

File: README.md (hunk 4 above)
Comment (toannvit) [reply README.md]: removed

File: README.md (hunk 3 above)
Comment (toannvit) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (toannvit) [reply README.md:44]: added instructions to create vc , did

**Review by namhoang1604 — APPROVED** [review]
LGTM

**Comment by toannvit** [comment]
Switch base to `uncefact/main`

**Pull request closed without merging**